asmcomp/selectgen.ml

```diff
@@ -793,22 +773,12 @@ method emit_expr_aux (env:environment) exp :
           self#insert_debug env  (Iraise k) dbg rd [||];
           None
       end
-<<<<<<< HEAD
   | Cop(Ccmpf _, _, dbg) ->
```

Comment:
just a note that the case of `Cop(Ccmpf _, _, dbg)`  doesn't exist in flambda-backend because we have already backported https://github.com/ocaml/ocaml/pull/9945.